CMakeLists.txt

```diff
@@ -360,6 +360,7 @@ list(REMOVE_ITEM __tmp_srcs ${onnx_gtests_src})
 list(APPEND ONNX_SRCS ${__tmp_srcs})
 
 add_library(onnx_proto ${ONNX_PROTO_SRCS} ${ONNX_PROTO_HDRS})
+add_dependencies(onnx_proto gen_onnx_operators_proto gen_onnx_data_proto)
```

Comment:
do you need to add gen_onnx_proto? If not why?

Reply:
Once you add `add_dependencies(onnx_proto gen_onnx_operators_proto)`
The `gen_onnx_data_proto` will be the next failure